x/perpetual/keeper/mtp_borrow_interest.go

```diff
@@ -82,17 +82,11 @@ func (k Keeper) SettleMTPBorrowInterestUnpaidLiability(ctx sdk.Context, mtp *typ
 		return math.ZeroInt(), err
 	}
 
-	revenueAmount := borrowInterestPaymentInCustody.Sub(takeAmount)
-	err = k.TransferRevenueAmount(ctx, revenueAmount, mtp.CustodyAsset, &ammPool)
-	if err != nil {
-		return math.ZeroInt(), err
-	}
-
 	if !takeAmount.IsZero() {
 		k.EmitFundPayment(ctx, mtp, takeAmount, mtp.CustodyAsset, types.EventIncrementalPayFund)
 	}
 
-	err = pool.UpdateCustody(mtp.CustodyAsset, borrowInterestPaymentInCustody, false, mtp.Position)
+	err = pool.UpdateCustody(mtp.CustodyAsset, takeAmount, false, mtp.Position)
```

Comment:
here the subtract should be borrowInterestPaymentInCustody as this is subtraction on global pool custody amount

Reply:
could you explain why it should be like that ?

we are not sending this amount from pool as earlier

Reply:
This is not updating in amm Pool, but perpetual global accounting pool

Reply:
updated